docs/current-1.1/tools/dotnet-nuget-delete.md

```diff
@@ -0,0 +1,81 @@
+---
+title: dotnet-nuget-delete command | .NET Core SDK
+description: The dotnet-nuget-delete command deletes or unlists a package from the server. 
+keywords: dotnet-nuget-delete, CLI, CLI command, .NET Core
+author: karann-msft
+ms.author: mairaw
+manager: wpickett
+ms.date: 11/10/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 6ddffde4-c789-4e90-990e-d35f6a6565d4
+---
+
+#dotnet-nuget-delete
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-delete -- Deletes or unlists a package from the server. 
```

Comment:
put dotnet-nuget-delete in a code block and remove the extra hyphen (similar to [dotnet-pack](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack))